### PR TITLE
JDK-8280950: RandomGenerator:NextDouble() default behavior non conformant after JDK-8280550 fix

### PR DESCRIPTION
**Pull request by jddarcy**
The original fix to JDK-8280550 contained a typo where r rather than bound was used as the first argument to nextAfter; this PR corrects that issue.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8280950](https://bugs.openjdk.java.net/browse/JDK-8280950): RandomGenerator:NextDouble() default behavior non conformant after JDK-8280550 fix

### Reviewers
 * [Brian Burkhalter](https://openjdk.java.net/census#bpb) (@bplb - **Reviewer**) ⚠️ Review applies to 2f25d031039b5c8397e1cc8434946704956e0777
 * [Jim Laskey](https://openjdk.java.net/census#jlaskey) (@JimLaskey - **Reviewer**) ⚠️ Review applies to 2f25d031039b5c8397e1cc8434946704956e0777

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7292/head:pull/7292` \
`$ git checkout pull/7292`

Update a local copy of the PR: \
`$ git checkout pull/7292` \
`$ git pull https://git.openjdk.java.net/jdk pull/7292/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7292`

View PR using the GUI difftool: \
`$ git pr show -t 7292`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7292.diff">https://git.openjdk.java.net/jdk/pull/7292.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/util/Random/RandomNextDoubleBoundary.java [hunk 1]
```diff
@@ -49,4 +55,37 @@ public static void main(String... args) {
...
+
+    public static void nextDoublesWithRange(double origin, double bound) {
+        RandomGenerator rg = new RandomGenerator() {
+                @Override
+                    public double nextDouble() {
+                    return Double.MAX_VALUE;
+                }
+
+                @Override
+                    public long nextLong() {
```
Comment (bplb) [inline test/jdk/java/util/Random/RandomNextDoubleBoundary.java:76]: Bad indentation.

File: test/jdk/java/util/Random/RandomNextDoubleBoundary.java [hunk 2]
```diff
@@ -49,4 +55,37 @@ public static void main(String... args) {
...
+                                      {5432167.234, 54321678.1238}};
+        for (double[] originAndBound : originAndBounds) {
+            nextDoublesWithRange(originAndBound[0], originAndBound[1]);
+        }
+    }
+
+    public static void nextDoublesWithRange(double origin, double bound) {
+        RandomGenerator rg = new RandomGenerator() {
+                @Override
+                    public double nextDouble() {
```
Comment (bplb) [inline test/jdk/java/util/Random/RandomNextDoubleBoundary.java:71]: Bad indentation.

**Review by bplb — APPROVED** [review]
Looks all right modulo the copyright year.

**Review by JimLaskey — APPROVED** [review]
LGTM

**Pull request closed without merging**